core/src/main/scala/chisel3/RawModule.scala

```diff
@@ -264,4 +268,28 @@ package object internal {
       }
     }
   }
+
+  /** Internal API for [[ViewParent]] */
+  sealed private[chisel3] abstract class ViewParentAPI extends RawModule()(ExplicitCompileOptions.Strict) {
+    private[chisel3] def fakeComponent: Component
+    private[chisel3] def absoluteTarget: IsModule = ModuleTarget(this.circuitName, "_$$AbsoluteView$$_")
```

Comment:
Why did you have to implement `absoluteTarget` but not `toTarget`?

Reply:
Note that this is not `overriding` `toAbsoluteTarget`, but rather providing an absolute target for use in `final def toAbsoluteTarget` in `BaseModule`. Explaining in a comment